dace/sdfg/validation.py

```diff
@@ -55,6 +58,46 @@ def validate_sdfg(sdfg: 'dace.sdfg.SDFG'):
                     "storage type. Please use a different storage location." %
                     name, sdfg, None)
 
+            #Check for valid bank assignments
+            try:
+                bankassignment = sdutil.parse_location_bank(desc)
```

Comment:
```suggestion
                bank_assignment = sdutil.parse_location_bank(desc)
```

Please go over the code and make sure you follow the formatting and naming conventions everywhere so I don't have to comment on each one :-)